src/pubkey.h

```diff
@@ -36,10 +36,10 @@ class CPubKey
     /**
      * secp256k1:
      */
-    static const unsigned int PUBLIC_KEY_SIZE = 65;
-    static const unsigned int COMPRESSED_PUBLIC_KEY_SIZE = 33;
-    static const unsigned int SIGNATURE_SIZE = 72;
-    static const unsigned int COMPACT_SIGNATURE_SIZE = 65;
+    static constexpr unsigned int PUBLIC_KEY_SIZE = 65;
+    static constexpr unsigned int COMPRESSED_PUBLIC_KEY_SIZE = 33;
+    static constexpr unsigned int SIGNATURE_SIZE = 72;
+    static constexpr unsigned int COMPACT_SIGNATURE_SIZE = 65;
```

Comment:
add 
```static constexpr unsigned int PUBLIC_KEY_HASH160_SIZE = 20;```
(and let's use it rather than assuming below)

Reply:
done